Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "UserReport",
+    platforms: [
+        .iOS(.v9)
+    ],
+    products: [
+        .library(
+            name: "UserReport",
+            targets: ["UserReportSDK"]
+        )
+    ],
+    targets: [
+        .target(
+            name: "UserReportSDK",
+            dependencies: [],
+            path: "UserReport/UserReport",
+            exclude: ["UserReport/UserReport/Info.plist"]
```

Comment:
Still seeing a warning from this unless I replace with just `Info.plist`, so I guess it should be relative to the above `path`:
```suggestion
            exclude: ["Info.plist"]
```